staging/src/k8s.io/apiserver/pkg/server/options/admission.go

```diff
@@ -94,10 +94,12 @@ func (a *AdmissionOptions) AddFlags(fs *pflag.FlagSet) {
 
 	fs.StringSliceVar(&a.EnablePlugins, "enable-admission-plugins", a.EnablePlugins, ""+
 		"admission plugins that should be enabled in addition to default enabled ones. "+
+		"(Default enabled plugins: "+strings.Join(a.defaultEnabledPluginNames(), ", ")+".) "+
```

Comment:
nit: `Foo bar bla. (Default enabled plugins: abc, def.)` looks strange typographically, doesn't it? Better like `... should be enabled in addition to default enabled ones (abc, def)`

Reply:
@sttts done.